### PR TITLE
fping: add page

### PR DESCRIPTION
**Pull request by jsonbruce**
A new command !

`fping` is a more powerful ping

### CODE REVIEW

File: pages/common/fping.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# fping
+
+> A more powerful ping which ping multiple hosts
```
Comment (waldyrious) [inline pages/common/fping.md]: "which ping" --> "which can ping". Also, you need a period at the end of the description.

File: pages/common/fping.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- List alive host within subnet via netmask
```
Comment (waldyrious) [inline pages/common/fping.md]: "host" --> "hosts"; I'd also suggest "via" --> "using a", which I think is clearer. By the way, does the `-g` stand for any word in particular?

File: pages/common/fping.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+`fping -a -g 192.168.1.0/24`
+
+- List alive host within subnet via ip range
```
Comment (waldyrious) [inline pages/common/fping.md]: Pluralize "host" here as well, and make "ip" uppercase. "Via" could also be replaced by "using a".

File: pages/common/fping.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+`fping -a -g 192.168.1.1 192.168.1.254`
+
+- List unreachable host within subnet
```
Comment (waldyrious) [inline pages/common/fping.md]: "hosts", as mentioned above.

Comment (jsonbruce) [inline pages/common/fping.md]: Thanks, I have pushed a new commit. By the way, `g` for Generate. @waldyrious

**Comment by waldyrious** [comment]
Thanks for the quick reaction! LGTM 👍

File: pages/common/fping.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+# fping
+
+> A more powerful ping which can ping multiple hosts.
+
+- List alive hosts within a given subnet using a netmask:
+
+`fping -a -g 192.168.1.0/24`
```
Comment (agnivade) [inline pages/common/fping.md]: What does `-g` mean ? Any way we can include it in the description ?

Comment (sbrl) [inline pages/common/fping.md]: "Generate list"?

Comment (jsonbruce) [inline pages/common/fping.md]: Yes, `g` for generate ! @agnivade 

Actually I have thought including generate in the description but I didn't get a comfortable way,
So, How do think the below

- List alive hosts within a **generated** subnet using a netmask

Comment (agnivade) [inline pages/common/fping.md]: How about `List alive hosts within a subnet generated using a netmask` ?

Comment (sbrl) [inline pages/common/fping.md]: Or even "Generate a list of alive hosts...."?

Comment (waldyrious) [inline pages/common/fping.md]: @sbrl the command does not generate the list of alive hosts: it generates a subnet, tests all hosts within it for aliveness, and returns the list of alive ones. I would rather go with something closer to @agnivade's proposal: `List alive hosts within a subnet generated from a netmask`.

**Pull request merged**